index.html

```
@@ -5896,6 +5896,18 @@ <h3>Inventory</h3>
                 </p>
                 <button class="btn theme-btn" id="forceRefreshBtn">Force Refresh</button>
               </div>
+
+              <!-- ── Data Reset ── -->
+              <div class="settings-fieldset">
+                <div class="settings-fieldset-title">Data Reset</div>
+                <p class="settings-subtext">
+                  Permanently remove inventory or wipe all app data. These actions cannot be undone.
+                </p>
+                <div class="settings-btn-row">
+                  <button class="btn warning" id="removeInventoryDataBtn">Remove Inventory</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)